docker/awscli.md

```diff
@@ -0,0 +1,29 @@
+Seekret
+=======
+Docker deployment  
+Sniffer docker will passively sniff target container traffic and move the pcaps to the remote bucket.
+Supported buckets: S3.
+
+The machine must be configured to have an IAM Role access or provide ACCESS_KEY and SECRET_KEY via the configuration to the docker  
```

Comment:
please mark this as bold or some other markdown magic

Reply:
done